### ISSUE TITLE
Define DKG error code

### ISSUE THREAD

**Issue by cmsigrist** [dedis/d-voting#139]
The DKG return the status Failed when the user tries to setup a node that is already Setup. Currently, the front-end checks during the polling the content of the error message and ignores the error if it contains the string "setup() was already called, only one call is allowed" (marking the Promise as resolved with the status Setup). 
An error code, corresponding to this case should be defined instead.

**Comment by nkcr** [comment]
True. The frontend can define (and document) one. The backend will then adapt.
